editor/modes/visual-editor/style.scss

```diff
@@ -255,3 +255,39 @@ $sticky-bottom-offset: 20px;
 		content: '';
 	}
 }
+
+.editor-visual-editor__placeholder {
```

Comment:
Are there common styles here we could extend from a base [class/SASS mixin/SASS placeholder] (or combine into a single selector) between the placeholder and actual blocks?